pom.xml

```diff
@@ -94,1841 +94,6 @@
     <!-- **************** -->
     <!-- Plugins versions -->
     <!-- **************** -->
-    <version.antrun.plugin>1.8</version.antrun.plugin>
```

Comment:
this deleted configuration will be inherited from meeds one ?

Reply:
no, ITOP copies the pom from addons-parent-pom
As currently, we are just creating the module, we let the dependencies in addons-parent-pom